src/Apps/Order/Routes/__tests__/Shipping.jest.tsx

```diff
@@ -1029,6 +1029,38 @@ describe("Shipping", () => {
 
           expect(mockCommitMutation).toHaveBeenCalled()
         })
+
+        describe("with address verfication enabled", () => {
+          beforeAll(() => {
+            ;(useFeatureFlag as jest.Mock).mockImplementation(
+              (featureName: string) => featureName === "address_verification_us"
+            )
+          })
+
+          afterAll(() => {
+            ;(useFeatureFlag as jest.Mock).mockReset()
+          })
+
+          it("triggers basic form validation first", async () => {
+            const wrapper = getWrapper({
+              CommerceOrder: () => testOrder,
+              Me: () => emptyTestMe,
+            })
+
+            const page = new ShippingTestPage(wrapper)
+
+            await page.clickSubmit()
+
+            const input = page
+              .find(Input)
+              .filterWhere(wrapper => wrapper.props().title === "Full name")
+            expect(input.props().error).toEqual("This field is required")
+
+            expect(
+              page.find(`[data-testid="address-verification-flow"]`).exists()
+            ).toBe(false)
```

Comment:
This test is able to [catch the issue](https://app.circleci.com/pipelines/github/artsy/force/50599/workflows/b13476cc-ea46-40d4-beae-edf02f111ad1/jobs/379197)! 🎉 

![rerun-causality-sync](https://github.com/artsy/force/assets/796573/e017a6a0-3048-4f23-9bfd-5310dd30c48a)